docs/assets/css/site.css

```diff
@@ -0,0 +1,520 @@
+/*
+    General Site Styling
+*/
+
+body {
+    background-color: #000;
+}
+
+h3 {
+    color: #fff;
+}
+
+p {
+    color: #fff;
+}
+
+a {
+    color: #b8ffd8;
+    transition: color 0.3s ease-in-out;
+}
+
+a:hover,
+a:visited {
+    color: #000;
+}
+
+.container.m-t-xl.m-b-xl {
+	background-color: #e5e5e5;
+    color: #fff;
+}
+
+/*
+    Page Layout
+*/
+
+.wrapper-content {
+    padding: 35px 20px 40px;
+}
+
+.sidebar {
```

Comment:
On small displays a side menu can be shown (using the green button in the bottom right corner of the page - not visible on landing page). Background color should be updated and it's top position should be 54px, seems to be a bit too low at its current position:

![image](https://user-images.githubusercontent.com/20795331/27143675-139ad868-5127-11e7-8d24-d8867725df95.png)